servers/zms/src/main/java/com/yahoo/athenz/zms/ZMSImpl.java

```diff
@@ -5172,11 +5177,12 @@ SignedDomain retrieveSignedDomainData(final String domainName, long modifiedTime
             domainData.setTokenExpiryMins(athenzDomain.getDomain().getTokenExpiryMins());
         }
 
-        // set the roles, services, and groups
+        // set the roles, services, groups and tags
 
         domainData.setRoles(athenzDomain.getRoles());
         domainData.setServices(athenzDomain.getServices());
         domainData.setGroups(athenzDomain.getGroups());
+        domainData.setTags(athenzDomain.getTags());
```

Comment:
to follow up from the other comment I would have expected here just to have:

domainData.setTags(athenzDomain.getDomain().getTags());